config/filters.json

```diff
@@ -21,5 +21,13 @@
         "checkObjectValues": true
       }
     }
+  },
+  {
+    "ext": ".yaml",
```

Comment:
It needs to be an array: `"ext": [".yaml"],` instead of `"ext": ".yaml",`, let's include another extension used by this format which is a shorter form `.yml`, that should be a new entry:

```
"ext": [".yaml", ".yml"],
```